requirements/base.txt

```diff
@@ -3,6 +3,7 @@ attrdict==2.0.1
 defusedxml==0.7.1
 einops==0.3.2
 FrEIA @ git+https://github.com/VLL-HD/FrEIA.git
+jsonargparse[signatures]
```

Comment:
I'm getting some warnings related to jsonargparse when I run `python trainer.py --help`. It works fine, but it would be better if it doesn't show the warnings. Maybe you could have a look.

```
/home/djameln/miniconda3/envs/anoseg/lib/python3.8/site-packages/jsonargparse/deprecated.py:36: JsonargparseDeprecationWarning: 
  By default only one JsonargparseDeprecationWarning per type is shown. To see
  all warnings set environment variable JSONARGPARSE_DEPRECATION_WARNINGS=all
  and to disable the warnings set JSONARGPARSE_DEPRECATION_WARNINGS=off.
  
  warning(
/home/djameln/miniconda3/envs/anoseg/lib/python3.8/site-packages/pytorch_lightning/utilities/cli.py:114: JsonargparseDeprecationWarning: 
  ``parse_as_dict`` parameter was deprecated in v4.0.0 and will be removed in
  v5.0.0. After removal, the parse_*, dump, save and instantiate_classes
  methods will only return Namespace and/or accept Namespace objects. If
  needed for some use case, config objects can be converted to a nested dict
  using the Namespace.as_dict method.